registration/tests/serializers/test_family_detail_update.py

```diff
@@ -0,0 +1,185 @@
+        }
+        self.parent = Student.objects.create(**self.parent_data)
+        self.family_data = {
+            "email": "weasleys@theorder.com",
+            "home_number": "123456789",
+            "cell_number": "144.618.8659x1124",
+            "work_number": "793.373.9334",
+            "preferred_number": "Work",
+            "address": "12 Grimmauld Place",
+            "preferred_comms": "Owl Post",
+            "parent": self.parent,
+            "notes": "Development perhaps successful they set.",
+        }
+        self.family = Family.objects.create(**self.family_data)
+        self.parent.family = self.family
+
+        self.children_data = [
+            {
+                "first_name": "Ron",
+                "last_name": "Weasley",
+                "family": self.family,
+                "role": Student.CHILD,
+                "date_of_birth": "2020-03-05",
+                "information": {f"{self.child_field.id}": "male"},
+            },
+            {
+                "first_name": "Ginny",
+                "last_name": "Weasley",
+                "family": self.family,
+                "role": Student.CHILD,
+                "date_of_birth": "2020-03-05",
+                "information": {f"{self.child_field.id}": "female"},
+            },
+        ]
+        self.guests_data = [
+            {
+                "first_name": "Harry",
+                "last_name": "Potter",
+                "family": self.family,
+                "role": Student.GUEST,
+                "date_of_birth": "2020-03-05",
+                "information": {f"{self.guest_field.id}": "friend"},
+            }
+        ]
+
+        child1 = Student.objects.create(**self.children_data[0])
+        child2 = Student.objects.create(**self.children_data[1])
+        guest = Student.objects.create(**self.guests_data[0])
+        self.children_data[0]["id"] = child1.id
+        self.children_data[1]["id"] = child2.id
+        self.guests_data[0]["id"] = guest.id
+        self.parent_data["id"] = self.parent.id
+        self.child = child1
+
+        self.family_data["children"] = self.children_data
+        self.family_data["guests"] = self.guests_data
+        self.family_data["parent"] = self.parent_data
+        self.family_data["current_enrolment"] = None
+
+    def test_family_detail_serializer_update(self):
+        data = dict(self.family_data)
+        data["email"] = "new_email"
+        data["home_number"] = "new_home_number"
+        data["preferred_comms"] = "Phone"
+
+        serializer = FamilyDetailSerializer(instance=self.family, data=data)
+        self.assertTrue(serializer.is_valid())
+        family = serializer.save()
+
+        self.assertEqual(family.email, data["email"])
+        self.assertEqual(family.home_number, data["home_number"])
+        self.assertEqual(family.preferred_comms, data["preferred_comms"])
+
+    def test_family_detail_serializer_update_child(self):
```

Comment:
nit: can we make these test names a tad more consistent? 
```suggestion
    def test_family_detail_serializer_update__update_children(self):
```